Don't redefine "CALL" macro

when compiling from source

```
In file included from /mnt/c/Users/xxx/AppData/Local/Temp/1309/sources/all.c:5:
/mnt/c/Users/xxx/AppData/Local/Temp/1309/sources/cosimulation.c:18: warning: "CALL" redefined
   18 | #define CALL(f) do { const Status status = f; if (status != OK) return status; } while (false)
      | 
In file included from /mnt/c/Users/xxx/AppData/Local/Temp/1309/sources/all.c:3:
/mnt/c/Users/xxx/AppData/Local/Temp/1309/sources/fmi2Functions.c:178: note: this is the location of the previous definition
  178 | #define CALL(f) do { \
      | 
```

--- a/src/cosimulation.c
+++ b/src/cosimulation.c
@@ -15,6 +15,10 @@
 #define strdup _strdup
 #endif
 
+#ifdef CALL
+#undef CALL
+#endif
+
 #define CALL(f) do { const Status status = f; if (status != OK) return status; } while (false)
 
 

